suite-native/module-send/src/screens/SendAccountsScreen.tsx

```diff
@@ -16,23 +13,25 @@ const BITCOIN_LIKE_FILTER = 'bitcoin';
 export const SendAccountsScreen = ({
     navigation,
 }: StackProps<SendStackParamList, SendStackRoutes.SendAccounts>) => {
+    const { translate } = useTranslate();
+
     const navigateToSendFormScreen: OnSelectAccount = ({ account }) =>
         navigation.navigate(SendStackRoutes.SendOutputs, {
             accountKey: account.key,
         });
 
     // TODO: move text content to @suite-native/intl package when is copy ready
     return (
-        <Screen
+        <SendScreen
             screenHeader={<DeviceManagerScreenHeader />}
-            subheader={<ScreenSubHeader content="Send from" leftIcon={<GoBackIcon />} />}
+            subheader={<SendScreenSubHeader content={translate('moduleSend.accountsList.title')} />}
```

Comment:
Since we have `screenHeader`, would this make sense?
```suggestion
            subHeader={<SendScreenSubHeader content={translate('moduleSend.accountsList.title')} />}
```